website/cue/reference/components/sources/opentelemetry.cue

```diff
@@ -32,8 +32,6 @@ components: sources: opentelemetry: {
 					ssl: "optional"
 				}
 			}
-			receive_buffer_bytes: enabled: false
-			keepalive: enabled:            true
```

Comment:
I believe these are old options from Vector v1 source, and were incorrectly copied over.